components/dashboard/public/index.html

```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script>
+      window.PREVIEW_URL = "%REACT_APP_PREVIEW_URL%";
+    </script>
```

Comment:
Let's not make this a variance. Just hard coding it temporarily seems fine and simple.